ProjectPolice/police.py

```diff
@@ -23,22 +29,32 @@ def publish_to_notifier(message, channel):
         body=payload,
         properties=pika.BasicProperties(delivery_mode=2),
     )
+    return {
+        "success": True,
+        "data": {
+            "message": "message sent to Notifier",
+            "resources": payload,
+        },
+    }
 
 
 # Function to send message to Project Microservice
 def send_to_projectms(message):
     milestone_id = message["data"]["milestone_id"]
     project_id = message["data"]["project_id"]
     payload = format_message(milestone_id, project_id, message["data"])
```

Comment:
The request body json payload to `PATCH /projects/:id/milestone/:mid/status` should be
```json
{
  "status": "met"
}
```